aopy/analysis/base.py

```diff
@@ -1621,3 +1621,42 @@ def calc_corr2_map(data1, data2, knlsz=15, align_maps=False):
     NCC[nan_idx1] = np.nan
     
     return NCC, shifts
+
+def get_neighboring_channel_index(elec_pos, distance=0.75):
```

Comment:
what do you think about having this function take "ch_idx" as an argument, and only return neighboring_idx[ch_idx], instead of the whole matrix?
i think it might be more useful that way? 
probably it will be slower, so in get_mean_neighboring_channels() you might not use this function